user-guide/Standard_Apps/EPM/EPM_I-DOCSIS/I-DOCSIS_parameters/I-DOCSIS_parameters_ptp.md

```diff
@@ -0,0 +1,29 @@
+---
+uid: I-DOCSIS_parameters_ptp
+---
+
+# I-DOCSIS parameters – PTP
+
+These parameters are only available in elements running the Harmonic CableOS connector, on the PTP page.
+
+Currently, these parameters are not shown on the topology chains available in I-DOCSIS. However, you can access this information using a dashboard, low-code app, or custom visual overview.
+
+- **CCAP PTP Clock State**: Direct value. This attribute represents the current state of the clock.
+
+  OID: 1.3.6.1.4.1.4491.2.1.32.1.1.2.1.1
+
+- **RPD Port PTP Status Table**
+
+  - **IP address type**: Direct value. This attribute represents the IP address type of the PTP master clock source that is currently being used.
+
+    OID: 1.3.6.1.4.1.4491.2.1.32.1.1.4.1.6
+
+  - **IP Address SNMP**: This attribute reports the IP address of the PTP master clock source that is currently being used.
+
+    OID: 1.3.6.1.4.1.4491.2.1.32.1.1.4.1.7
+
+- **RPD PTP Clock Status Table**
+
+  - **Clock State**: Direct value. This attribute represents the current state of the clock for the instance.
+
+    OID: 1.3.6.1.4.1.4491.2.1.32.1.1.2.1.1
```

Comment:
Hi @sergiosuarezsky, can we add a note here that indicates that these KPIs are targeted to a distributed architecture (D-DOCSIS). Currently we are adding these KPIs in the connector since this connector will also be used in the D-DOCSIS solution.